File: Leetcode Easy/1480_Running Sum Array.java
class Solution {
    public int[] runningSum(int[] nums) {
        // initially the sum will be 0
        int sum = 0;
        // result array
        int[] result = new int[nums.length];
        for (int i = 0; i < nums.length; i++) {
            // sum = sum+nums[i]
            sum += nums[i];
            // push sum in result
            result[i] = sum;
        }
        return result;
    }
}

File: Leetcode Easy/1480_Running Sum Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */

var runningSum = function (nums) {
    //Initialy sum will be zero
    let sum = 0;
    //result array
    let result = [];

    for (let i = 0; i < nums.length; i++) {
        //sum = sum+nums[i]
        sum += nums[i];
        //push sum in result
        result.push(sum)

    }
    
    return result;

};